test/integration/tcp_proxy_integration_test.cc

```diff
@@ -434,8 +523,13 @@ TEST_P(TcpProxyIntegrationTest, AccessLog) {
     access_log_config.set_path(access_log_path);
     access_log_config.mutable_log_format()->mutable_text_format_source()->set_inline_string(
         "upstreamlocal=%UPSTREAM_LOCAL_ADDRESS% "
-        "upstreamhost=%UPSTREAM_HOST% downstream=%DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT% "
-        "sent=%BYTES_SENT% received=%BYTES_RECEIVED%\n");
+        "upstreamhost=%UPSTREAM_HOST% "
```

Comment:
Since you added a helper which is used for similar reasons as the code above, can you invoke your helper here?

Reply:
Fixed.